### DISCUSSION TITLE
Is there a simple way to make the page multilingual?

### DISCUSSION THREAD

**Discussion by sacacitas** [GetPublii/Publii#1566] [Help]
I saw there is some pages with subdirectories /en /es /de on Publii showcase section. Does somebody know how they do it?

**Comment by candidexmedia** [comment]
Bob mentioned in the old forum back in January that [a multilingual plugin is on the horizon](https://forum.getpublii.com/topic/multilanguage-support/#post-11556), but no specific release date yet.

For now, automated translations via a plugin or site mods are possible. See my gist: [Publii-Google-Translate-Widget-HTML.md ](https://gist.github.com/candidexmedia/03c0b74ca88a4bdec06cf0148d6758ad)

**Comment by growdigital** [comment]
Really looking forward to a multilingual plugin. I'm working with a network of NGOs and I have **25** potential customers if there was a way to do a multilingual site easily! Thank you for a great project :)
